ios/RNSScreenWindowTraits.mm

```diff
@@ -44,11 +44,11 @@ + (UIStatusBarStyle)statusBarStyleForRNSStatusBarStyle:(RNSStatusBarStyle)status
     __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0
   if (@available(iOS 13.0, *)) {
     switch (statusBarStyle) {
-      case RNSStatusBarStyleAuto:
+      case RNSStatusBarStyleSystem:
```

Comment:
What if someone passes `RNSStatusBarStyleAuto` now?

Reply:
Hmm, since we're not controlling the behavior of `auto` prop on the native side (it's handled on JS with the `dark` prop from useTheme()), I believe we don't need to handle the case of `auto` here.